qemu/tests/timedrift_with_reboot.py

```diff
@@ -28,7 +29,13 @@ def run(test, params, env):
                                       args_added=boot_option_added)
 
     timeout = int(params.get("login_timeout", 360))
-    session = vm.wait_for_login(timeout=timeout)
+
+    if params.get("os_type") == 'linux':
+        session = vm.wait_for_login(timeout=timeout)
+        utils_time.sync_timezone_linux(session=session)
+    elif params.get("os_type") == 'windows':
+        utils_time.sync_timezone_win(vm)
```

Comment:
I read the code and found that there is a relaunch operation of the vm within the function, so would you please help explain why we should have it, I think it is the reason about why you added these `if .. elif ..` clause at here.

Reply:
@luckyh Thanks for review, it is because for windows guest, time zone set will not take effect unless we boot it again from qemu commands, and for linux guest, no need to boot it again in our test, so I add clause here. Thanks.

Reply:
@sitoliu does it take effect with a reboot? I mean for windows.

Reply:
@luckyh Nope.

Reply:
@sitoliu ok, thanks for your explanation, then based the fact, I'd suggest to have the same api (by taking `vm` as the argument) for linux and windows as @hereischen said, thus, we can move `session = vm.wait_for_login(timeout=timeout)` out from the if clause, thanks.